src/features/saved-questions/ui/SavedQuestionsPage.tsx

```
@@ -263,9 +267,10 @@ const QuestionViewDialog: React.FC<QuestionViewDialogProps> = ({
               </CardHeader>
               <CardContent>
                 <div className="p-4 bg-blue-50 border-l-4 border-blue-400 rounded-r-lg">
-                  <div className="text-sm leading-relaxed whitespace-pre-wrap text-blue-900">
-                    {question.note}
-                  </div>
+                  <div 
+                    className="text-sm leading-relaxed text-blue-900"
+                    dangerouslySetInnerHTML={{ __html: renderFormattedNote(question.note || '') }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)